### PR TITLE
Add optional social question to stand-up

### LINKED ISSUE #528: Add "question of the day" suggestion and examples
We had excellent team cohesion in an engagement after introducing a short question of the day (e.g. "what's your favorite fast food" or "would you rather live underwater or in space for a year") during stand ups. Over the course of a month this introduced personality context, lowering the barrier to reaching out when stuck. 

The [stand ups](https://github.com/microsoft/code-with-engineering-playbook/tree/master/agile-development/stand-ups) page could include a suggestion to try such and a list of examples.

### PR DESCRIPTION
**Pull request by cdw**
Describes the purpose and implementation of an optional social question to follow the three project questions during the daily stand-up. Closes #528 .

### CODE REVIEW

File: agile-development/stand-ups/readme.md [hunk 1]
```diff
@@ -78,12 +80,16 @@ If a contributor is not working on an existing sprint task they need to either c
...
 
+### Social question
+
+Teams are frequently geographically distributed and include members who have not worked on projects together previously. Small social interactions facilitate interpersonal trust development and lower the barriers to collaboration. A social question-of-the-day that has a one-sentence answer contributes to trust development over the course of many stand-ups, with a minimal additional time commitment. The answer to the social question should be brief and follow the project questions answer. The facilitator may choose the social question or take a suggestion from the room. Description of what makes a good question and a list of starter questions are available within the [social question readme](./social-question/readme.md).
+
 ### Start On Time
 
 Make a best effort to begin answering the 3 questions as close to the scheduled start time as possible. Try not to waste time upfront on chit-chat or waiting on all team members to join. This can extend the meeting time significantly. Starting immediately will help ensure stand-ups remain effective and useful over time.
```
Comment (fnocera) [inline agile-development/stand-ups/readme.md:95]: Typo: "3" should be removed

File: agile-development/stand-ups/readme.md [hunk 2]
```diff
@@ -78,12 +80,16 @@ If a contributor is not working on an existing sprint task they need to either c
...
 
 > Parking lot discussions are optional for participants.
 
 Ensure discussion leaders call out necessary parties for their discussion points upfront, allowing those not needed to leave the meeting.
 
+### Social question
+
+Teams are frequently geographically distributed and include members who have not worked on projects together previously. Small social interactions facilitate interpersonal trust development and lower the barriers to collaboration. A social question-of-the-day that has a one-sentence answer contributes to trust development over the course of many stand-ups, with a minimal additional time commitment. The answer to the social question should be brief and follow the project questions answer. The facilitator may choose the social question or take a suggestion from the room. Description of what makes a good question and a list of starter questions are available within the [social question readme](./social-question/readme.md).
```
Comment (fnocera) [inline agile-development/stand-ups/readme.md:91]: Nit: would rephrase to "Social interactions facilitate the development of trust between team members and lowers the barriers for collaboration". Small social interactions sounded a little funny.

File: agile-development/stand-ups/social-question/readme.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Social question of the day
+
+The social question of the day is an optional short question to follow the [three project questions](../readme.md) in the daily stand-up. It develops team cohesion and interpersonal trust over the course of an engagement by facilitating the sharing of chosen personal context.
```
Comment (fnocera) [inline agile-development/stand-ups/social-question/readme.md:3]: Nit: "sharing of chosen personal context" consider replacing with "sharing of personal preferences, lifestyle or other context".

File: agile-development/stand-ups/social-question/readme.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+If the team decides to incorporate a social question-of-the-day into the stand-up, the questions can be selected by the facilitator or sourced from the attendees. The latter is faster and the former will likely result in more diverse questions.
```
Comment (fnocera) [inline agile-development/stand-ups/social-question/readme.md:5]: It would be best to suggest that the facilitator should select the question ahead of time as it makes it more efficient. I know we have not been doing this recently but we are in a small group at the moment. 
I would also change this guidance in the main social question section above.

File: agile-development/stand-ups/social-question/readme.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+> Tip: having the stand-up facilitator role rotate lets the facilitator choose the social question without burdening any one team member.
```
Comment (fnocera) [inline agile-development/stand-ups/social-question/readme.md:7]: would add "facilitator role rotate **each sprint** lets the ..."

**Review by fnocera — APPROVED** [review]
Thanks for the contribution - this was a great addition during our engagement so I hope it's something that other teams can benefit from too!

File: agile-development/stand-ups/social-question/readme.md (hunk 5 above)
Comment (cdw) [reply agile-development/stand-ups/social-question/readme.md]: Agreed, changed.

File: agile-development/stand-ups/social-question/readme.md (hunk 4 above)
Comment (cdw) [reply agile-development/stand-ups/social-question/readme.md]: Agreed that the question should be selected ahead of time. I changed it to suggest either independent selection by the facilitator or selection from some async suggestions. What do you think?

File: agile-development/stand-ups/readme.md (hunk 1 above)
Comment (cdw) [reply agile-development/stand-ups/readme.md]: A few other places too! 👍

File: agile-development/stand-ups/social-question/readme.md (hunk 4 above)
Comment (fnocera) [reply agile-development/stand-ups/social-question/readme.md]: perfect!

**Review by KSLHacks — APPROVED** [review]
Thanks for putting this together David! The PR looks good, as a member on the project I really felt that this was the "next best" alternative to fill the gap on the team building and camaraderie we usually build during our on-site visits with the teams.

"reduce social barriers to collaboration" resonated with me.
